dapr_agents/workflow/orchestrators/llm/orchestrator.py

```diff
@@ -136,48 +241,58 @@ def main_workflow(self, ctx: DaprWorkflowContext, message: TriggerAction):
             step_id = next_step.get("step", None)
             substep_id = next_step.get("substep", None)
 
-            # Step 5: Validate Step Before Proceeding
+            if not ctx.is_replaying:
```

Comment:
this section seems strange, or maybe was during debug? as plan_dicts is not used anymore in this method I think